kaska/s2_observations.py

```diff
@@ -29,17 +29,30 @@ def __init__(
         parent_folder,
         emulator,
         state_mask,
-        band_prob_threshold=20,
+        band_prob_threshold=5,
         chunk=None,
-        time_grid=None
+        time_grid=None,
     ):
         self.band_prob_threshold = band_prob_threshold
         parent_folder = Path(parent_folder)
         if not parent_folder.exists():
-            LOG.info(f"S2 data folder: {parent_folder}")
+            LOG.info(f"S2 data folder: {str(parent_folder):s}")
```

Comment:
I believe, `str` and `:s` are doing the same, I would use `:s`. However, in a `Path` it seems to not work,  but these objects should not need `str` either.
```pyhon
>>> from pathlib import Path
>>> a = Path('.')
>>> a
PosixPath('.')
>>> f"blalal {a}"
'blalal .'
>>> f"blalal {a:s}"
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
TypeError: unsupported format string passed to PosixPath.__format__
```

Reply:
As formatting with neither `str()` or `:s` seems to work, I'll use that.